tests/foreman/ui/test_dashboard.py

```diff
@@ -0,0 +1,485 @@
+
+    @stubbed()
+    @tier3
+    def test_positive_latest_warning_error_tasks(self):
+        """@Test: Check if Latest Warning/Error
+        Tasks widget is working in Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Review the Latest Warning/Error Tasks widget.
+
+        @Assert: The links to all failed/warnings tasks are working
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_content_view_history(self):
+        """@Test: Check if Content View History
+           widget is working in Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Review the Content View History widget
+
+        @Assert: Each Content View link shows its current status
+        (the environment to which it is published)
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_discovered_hosts(self):
+        """@Test: Check if user can access
+           Discovered Hosts widget
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Review the Discovered Hosts widget
+        3.Click on the list of Discovered Hosts
+
+        @Assert: It takes you to discovered hosts
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_latest_event_widget(self):
+        """@Test: Check if Latest Event
+           widget is working in Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Review the Latest Events widget
+
+        @Assert: The Widget is updated with
+        all the latest events
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_sync_overview_widget(self):
+        """@Test: Check if Sync Overview
+           widget is working in Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Create a product
+        2.Add a repo and sync it
+        3.Navigate to Monitor -> Dashboard
+        4.Review the Sync Overview widget
+        for the above sync details
+
+        @Assert: Sync Overview widget is
+        updated with all sync processes
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_content_host_subscription_status(self):
+        """@Test: Check if Content Host Subscription Widget
+           widget is working in Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Register Content Host and subscribe it
+        2.Navigate Monitor -> Dashboard
+        3.Review the Content Host Subscription Status
+        4.Click each link :
+          a.Invalid Subscriptions
+          b.Insufficient Subscriptions
+          c.Current Subscriptions
+
+        @Assert: The widget is updated with
+        all details for Current,
+        Invalid and Insufficient Subscriptions
```

Comment:
The assert can fit in two lines.  Remember each line can contain upto 79 characters.